source/main/windows/main.js

```diff
@@ -5,14 +5,14 @@ import ipcApi from '../ipc-api';
 import { runtimeFolderPath } from '../config';
 
 export const createMainWindow = () => {
-  var params = {
+  const params = {
```

Comment:
i thought const would prevent further changes to the object?

Reply:
@cleverca22 In JavaScript, `const` means that the identifier can’t be reassigned. (Not to be confused with immutable values. Unlike true immutable datatypes such as those produced by Immutable.js and Mori, a `const` object can have properties mutated.)

Reply:
ahh